### ISSUE TITLE
honor gulp --silent mode

### ISSUE THREAD

**Issue by rafegoldberg** [schickling/gulp-webserver#123]
yo, so first off this `gulp-webserver` guy is a fantastic ‘lil plugin. I've only got one (minor!) bone to pick, which is that the plugin continues to print logs even when I've run my task in silent mode... (I tried passing the `--silent` flag at the command line level, as well as setting a default in my `.gulprc` file — neither had any effect). Anyways, I read through the docs and a couple of issues, but I didn't see anything quite related...

In short, I guess I'm wondering if there's an easy way to silence _all_ logs from this plugin? Thanks in advance!

**Comment by hyeri0609** [comment]
What kind of log do you get? May i see?

**Comment by rafegoldberg** [comment]
I still get the default log line:

```
[11:31:38] Webserver started at http://0.0.0.0:8080
```
